test/integration/generated/mediaTypesRest/src/parameters.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { RequestParameters } from "@azure-rest/core-client";
+import { SourcePath } from "./models";
+
+export interface AnalyzeBodyBodyParam {
+  /**
+   * Input parameter.
+   *
+   * Value may contain any sequence of octets
+   */
+  body?: string;
+}
+
+export interface AnalyzeBodyMediaTypesParam {
+  /** Request content type */
+  contentType?: "application/pdf" | "image/jpeg" | "image/png" | "image/tiff";
```

Comment:
Will we pass the value for the customer by default if this contentType is a single value but not equals to application/json ?

Reply:
Nope, we can't pass values by default as we are not generating any runtime code, just type definitions. We could explore making contentType required and only accept one contentType. But I would like to wait until we get feedback on that. In the meantime we'll keep it optional and default to json

Reply:
I see. Thanks